steps/household/rent.tsx

```diff
@@ -316,8 +307,8 @@ const step: ProcessStepDefinition = {
             >;
           }): boolean {
             return (
-              stepValues["rent-arrears-type"] ===
-              ("yes has plan" || "yes has no plan")
+              stepValues["rent-arrears-type"] === "yes has plan" ||
+              stepValues["rent-arrears-type"] === "yes has no plan"
```

Comment:
Do you think it'd be worth creating a function that returns this conditional rather than duplicating it so many times?

Reply:
Probably a good idea.